src/test/java/org/rulez/demokracia/pdengine/ChoiceDeleteValidationTest.java

```diff
@@ -0,0 +1,68 @@
+package org.rulez.demokracia.pdengine;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.rulez.demokracia.pdengine.annotations.tested_behaviour;
+import org.rulez.demokracia.pdengine.annotations.tested_feature;
+import org.rulez.demokracia.pdengine.annotations.tested_operation;
+import org.rulez.demokracia.pdengine.exception.ReportedException;
+import org.rulez.demokracia.pdengine.testhelpers.CreatedDefaultVoteRegistry;
+
+public class ChoiceDeleteValidationTest extends CreatedDefaultVoteRegistry {
+
+	public ChoiceDeleteValidationTest() {
+		super();
+	}
+
+	@tested_feature("Manage votes")
+	@tested_operation("delete choice")
+	@tested_behaviour("deletes the choice")
```

Comment:
it is @tested_behaviour("validates inputs")
[Deviation in model](http://adadocs.demokracia.rulez.org/PDEngine/edemo/master/index.html#9386718b-3caf-4152-8386-0df9deeb35d3-3044c21c-7606-4098-8176-9dd5b93c20ad-af499e94-b21e-4295-8f22-efe9e9e0be15)

Reply:
Fixed